src/run.js

```diff
@@ -1,25 +1,27 @@
 'use strict';
 
-const { promisify } = require('util');
-const { exec, spawn } = require('child_process');
+const { spawn } = require('child_process');
 const debug = require('debug')('git-diff-apply');
-const execPromise = promisify(exec);
 
-module.exports = async function run(command, options) {
+const run = async function run(command, options = {}) {
+  // Matches the shell-enabled behaviour of child_process.exec, which this
+  // function previously used to run the specified command. This avoids
+  // problems when handling quotes and whitespace.
+  options.shell = true;
   debug(command);
-  let { stdout } = await execPromise(command, options);
+  let [cmd, ...args] = command.split(' ');
```

Comment:
Will `--foo="bar baz"` still get processed correctly with this split?

Reply:
Hmmm I believe it won't. It'll split inside the quotes.

@DeclanBoller we'll need to take another pass at this, improving the split logic to handle this case.